### PR TITLE
HDDS-5567. interface/Cli.md translation

### PR DESCRIPTION
**Pull request by will-sh**
## What changes were proposed in this pull request?
HDDS-5567. interface/Cli.md translate to Chinese.

Please describe your PR in detail:
Translated interface/Cli.md to Chinese.
Corrected a parent title from "编程接口" to "客户端接口" to better align with the English expression "Client Interfaces"

## What is the link to the Apache JIRA

[HDDS-5567](https://issues.apache.org/jira/browse/HDDS-5567)

## How was this patch tested?
hugo serve

### CODE REVIEW

**Comment by will-sh** [comment]
corrected a word from "垃圾回收站" to "回收站"

File: hadoop-hdds/docs/content/interface/Cli.zh.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+Ozone shell 是从命令行与 Ozone 交互的主要接口。在后台，它调用 [Java API]({{< ref "interface/JavaApi.md">}}).
+
+有些功能只能通过使用 `ozone sh` 命令才能访问。例如：
+
+1. 创建带有配额的卷
+2. 管理内部 ACL
+3. 创建带有加密的键的桶
+
+所有这些命令都是一次性的管理任务。应用程序也可以使用其他接口，如 Hadoop 兼容文件系统（o3fs 或 ofs）或 S3 接口来实现相同功能而无需使用Ozone命令行接口。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Cli.zh.md:34]: nit: space around words

```suggestion
所有这些命令都是一次性的管理任务。应用程序也可以使用其他接口，如 Hadoop 兼容文件系统（o3fs 或 ofs）或 S3 接口来实现相同功能而无需使用 Ozone 命令行接口。
```

File: hadoop-hdds/docs/content/interface/Cli.zh.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+这条命令输出卷的`create`动作的所有命令行选项
+
+## 通用命令格式
+
+Ozone shell 命令采取以下形式:
+
+> _ozone sh object action url_
+
+**ozone** 脚本用于调用所有 Ozone 子命令。通过 ```sh``` 命令调用 ozone shell命令。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Cli.zh.md:61]: nit
```suggestion
**ozone** 脚本用于调用所有 Ozone 子命令。通过 ```sh``` 命令调用 ozone shell 命令。
```

File: hadoop-hdds/docs/content/interface/Cli.zh.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+
+这条命令展示了卷的所有可用的 _动作_ 命令
+
+或者也可以用来解释具体的某个 _动作_ ，例如：
+
+```bash
+ozone sh volume create --help
+```
+
+这条命令输出卷的`create`动作的所有命令行选项
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Cli.zh.md:53]:
```suggestion
这条命令输出卷的 `create` 动作的所有命令行选项
```

File: hadoop-hdds/docs/content/interface/Cli.zh.md [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+根据动作，Ozone URL 可以指向以下格式的卷、桶或键:
+
+_\[schema\]\[server:port\]/volume/bucket/key_
+
+
+其中，
+
+1. **Schema** - 应为 `o3`，这是访问 Ozone API 的原生 RPC 协议。是否指定 schema 是可选的。
+
+2. **Server:Port** - 应为Ozone Manager的地址。如果不指定端口，则将使用 ozone-site.xml 中的默认端口。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Cli.zh.md:74]:
```suggestion
2. **Server:Port** - 应为 Ozone Manager 的地址。如果不指定端口，则将使用 ozone-site.xml 中的默认端口。
```

File: hadoop-hdds/docs/content/interface/Cli.zh.md [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+This command will delete volume recursively.
+There is no recovery option after using this command, and no trash for FSO buckets.
+Delay is expected running this command.
+Enter 'yes' to proceed': yes
+Volume vol1 is deleted
+```
+
+## 桶操作
+
+桶是层次结构的第二层级，与AWS S3桶相似。如果用户有必要的权限，可以在卷中创建桶。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Cli.zh.md:157]:
```suggestion
桶是层次结构的第二层级，与 AWS S3 桶相似。如果用户有必要的权限，可以在卷中创建桶。
```

File: hadoop-hdds/docs/content/interface/Cli.zh.md [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+```
+如果键是在OBS桶中，它将被永久删除。
+
+## 查询命令行结果
+
+Ozone命令行返回JSON响应。[jq](https://stedolan.github.io/jq/manual/) 是一个命令行JSON处理器，可以用来过滤CLI结果以获取所需信息.
+
+示例命令:
+
+* 列出不是链接的FSO桶。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Cli.zh.md:257]:
```suggestion
* 列出不是链接的 FSO 桶。
```

File: hadoop-hdds/docs/content/interface/Cli.zh.md [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+  "modificationTime": "2023-02-01T05:18:46.974Z",
+  "quotaInBytes": -1,
+  "quotaInNamespace": -1,
+  "bucketLayout": "FILE_SYSTEM_OPTIMIZED",
+  "owner": "om",
+  "link": false
+}
+```
+
+* 列出EC桶以及它们的复制策略配置。
```
Comment (smengcl) [inline hadoop-hdds/docs/content/interface/Cli.zh.md:278]:
```suggestion
* 列出 EC 桶以及它们的复制策略配置。
```

**Review by smengcl — APPROVED** [review]
Thanks @will-sh . lgtm apart from minor nits

**Review by smengcl — APPROVED** [review]

**Pull request merged**